### ISSUE TITLE
Some Perspectives on the Relationship between R-CoT's Method and Reversal of Thought

### ISSUE THREAD

**Issue by panhongai66** [dle666/R-CoT#2]
It is quite interesting that although this is multimodal rather than single-text modal, one can't help but wonder if the idea is inspired by  **Reversal of Thought**. 
I am very grateful to the author for creating a dataset and conducting many experiments and some open-source work. However, the overall approach seemingly borrows from **Reversal of Thought**. If there is a referenced idea, I think this work is also a reaffirmation of **Reversal of Thought**. It is advisable for authors to cite relevant works to clarify the motivation. https://arxiv.org/abs/2410.12323

**Comment by Yuliang-Liu** [comment]
Thank you for your attention and suggestions! We will include a discussion about this article in our next update. 
Here are few things we want to clarify:
1. Our submission predates the release of this article on October 16th, and we began working on our research well before then. 
![image](https://github.com/user-attachments/assets/ab46a171-1684-4178-81d5-622d6d221e12)

2. While the two works may share similar abbreviation, there are significant differences in their primary ideas. Our R-CoT approach focuses mainly on the data generation process, where we generate the answers first and then formulate the questions. This method helps to reduce inaccuracies caused by overly complex questions. On the other hand, the Reversal of Thought approach seems to emphasize optimizing prompts during the reasoning process to enhance the quality of the reasoning outcome.

We will make sure to clearly distinguish between these concepts in the updated version to avoid any confusion.

**Comment by panhongai66** [comment]
Thx for ur reply! **I noticed R-CoT (https://arxiv.org/html/2410.17885v1) was submitted on Oct. 23,** 
<img width="689" alt="image" src="https://github.com/user-attachments/assets/427a3088-6949-449a-be69-93de77780d31">
**while Reversal of Thought (https://arxiv.org/html/2410.12323v1) was on Oct. 16.** <img width="610" alt="image" src="https://github.com/user-attachments/assets/ec1e87f9-edb3-420f-a675-07ed7273db9e"> 
It's a bit earlier, so it may look like concurrent work. Noting recent work can strengthen the paper's focus on reverse reasoning logic methods and transition to R-CoT’s multimodal dataset. Looking forward to ur updated version!  Appreciate ur insights as I'm still learning. 😊 Thx again!

**Comment by Yuliang-Liu** [comment]
We have submitted to OpenReview on October 13, with the ArXiv version to follow later. BTW, I just found other papers with a similar abbreviation, just for your reference~ It seems like name duplication happens a lot :)

![image](https://github.com/user-attachments/assets/47dccbff-2cbf-4784-b658-3cb2a5ac0651)
![image](https://github.com/user-attachments/assets/7a28c375-8034-4ed4-b8b7-e41a4b4f3d93)
![image](https://github.com/user-attachments/assets/2f79c970-e5aa-436e-8475-4dae65abf42b)

I'll let you know once we've made the updates. Thanks again!
